sourced/ml/mining/__main__.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import logging
+import sys
+
+from modelforge import slogging
+
+from sourced.ml.mining.cmd import ArgumentDefaultsHelpFormatterNoNone
+
+
+def get_parser() -> argparse.ArgumentParser:
+    """
+    Create the cmdline argument parser.
+    """
+    parser = argparse.ArgumentParser(
+        formatter_class=ArgumentDefaultsHelpFormatterNoNone
+    )
+    parser.add_argument(
```

Comment:
Sorry, I missed this: can you please use [`add_logging_args`](https://github.com/src-d/modelforge/blob/master/modelforge/slogging.py#L228) - as a bonus, it will automatically set the logging level so that you don't have to call `slogging.setup()` yourself.

Reply:
Done, however in some case we need the log_level so I will not use the erase_arg option - however will remove manually everything, wyt ?